logstash-core-event-java/src/main/java/com/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -140,6 +132,8 @@ public IRubyObject ruby_set_field(ThreadContext context, RubyString reference, I
                     this.event.setField(r, RubyToJavaConverter.convertToList((RubyArray) value));
                 } else if (value instanceof RubyHash) {
                     this.event.setField(r, RubyToJavaConverter.convertToMap((RubyHash) value));
+                } else if (value instanceof RubyBoolean) {
+                    this.event.setField(r, ((RubyBoolean) value).toJava(boolean.class));
                 } else if (value.isNil()) {
                     this.event.setField(r, null);
                 } else {
```

Comment:
yeah, so what value type would you extract that to? for RubyInteger we use `.getLongValue())` for RubyFloat `getDoubleValue()` ... ?

Unless I missed something, I think we need to do each type independently.  

There shouldn't be a case where a Ruby symbol is in the Event data, if it is, its a bug no? but instead of crashing we could transform them to string maybe?

RubyBigDecimal does not exists?

I wonder if we shouldn't just do a Numeric catch all with:
```ruby
if (value instanceof RubyNumeric) {
  this.event.setField(r, ((RubyNumeric) value).toJava())
}
```